code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -502,7 +503,10 @@
 			"You feel like telling a pun.")
 			to_chat(M, "<span class='warning'>[pick(clown_message)]</span>")
 		if(M.mind.assigned_role == "Mime")
-			update_flags |= M.adjustToxLoss(1.5 * REAGENTS_EFFECT_MULTIPLIER)
+			if(!(M.dna.species.reagent_tag & PROCESS_ORG)) // Only activates if the mob that is processing the reagent is Synthetic (IPC)
```

Comment:
IMO, this check should be based on tox modifier
```suggestion
			if(!M.dna.species.tox_mod)) // If they can't take tox damage, make them take burn damage
```

Reply:
Seems the change caused 8/11 failed checks.

Reply:
There was a double parenthesis, this should fix it
```suggestion
			if(!M.dna.species.tox_mod) // If they can't take tox damage, make them take burn damage
```